aioftp/server.py

```diff
@@ -799,7 +805,8 @@ def __init__(self,
                  read_speed_limit_per_connection=None,
                  write_speed_limit_per_connection=None,
                  data_ports=None,
-                 encoding="utf-8"):
+                 encoding="utf-8",
+                 ssl=None):
```

Comment:
Pro tip: keep closing bracket on a separate line + use trailing commas — this makes diffs nicer.

Reply:
Personaly I use this code style, but now I use code style of this project...